Reference/conformance/conformance.pb.swift

```diff
@@ -88,43 +94,12 @@ enum Conformance_WireFormat: SwiftProtobuf.Enum {
     }
   }
 
-  init?(jsonName: String) {
-    switch jsonName {
-    case "UNSPECIFIED": self = .unspecified
-    case "PROTOBUF": self = .protobuf
-    case "JSON": self = .json
-    default: return nil
-    }
-  }
-
-  init?(protoName: String) {
```

Comment:
Can you file an issue describing your use case in detail?

Before the 1.0 milestone we're trying to lock down as much API as we can; it's much easier to add things later than it is to remove or change things once users depend on them. Just as we don't expose the field names as strings publicly, we want to discourage people depending on conversions between the raw enum names and the actual Swift values from outside the standard text format and JSON encoding/decoding processes.